### PR TITLE
Deuteronomy upgrade

### PR DESCRIPTION
**Pull request by EyeOfPython**

### CODE REVIEW

**Comment by maff1989** [comment]
Thank you @EyeOfPython! ❤️

File: src/chainparams.cpp [hunk 1]
```diff
@@ -111,6 +111,8 @@ class CMainParams : public CChainParams {
         consensus.numbersActivationTime = 1671659280;
         // 2023-06-21T14:58:00.000Z protocol upgrade
         consensus.deuteronomyActivationTime = 1687359480;
+        // 2023-12-22T03:27:00.000Z protocol upgrade
+        consensus.joshuaActivationTime = 1703215620;
```
Comment (schancel) [inline src/chainparams.cpp:115]: 👍  Verified

File: src/validation.cpp [hunk 2]
```diff
@@ -278,7 +278,7 @@ bool CheckSequenceLocks(const CTxMemPool &pool, const CTransaction &tx,
 static bool IsReplayProtectionEnabled(const Consensus::Params &params,
                                       int64_t nMedianTimePast) {
     return nMedianTimePast >= gArgs.GetArg("-replayprotectionactivationtime",
-                                           params.deuteronomyActivationTime);
+                                           params.joshuaActivationTime);
```
Comment (schancel) [inline src/validation.cpp:281]: 👍

**Comment by schancel** [comment]
https://github.com/givelotus/lotusd/wiki/Upgrade-Checklist

**Review by schancel — APPROVED** [review]

**Pull request merged**